config/config_mgmt.py

```diff
@@ -156,6 +156,24 @@ def sysLog(self, logLevel=syslog.LOG_INFO, msg=None, doPrint=False):
 
         return
 
+    def _filterTablesInConfig(self, config):
+        """
+            Check if any table should not be considered in Input Config, remove
+            such table from Input Config.
+            Parameters:
+                config(dict): Input Config
+            Returns:
+                void
+        """
+        # nonConfigTables should not be considered in Input Config.
+        nonConfigTables = ['BREAKOUT_CFG']
```

Comment:
Based on https://github.com/Azure/sonic-buildimage/pull/6801#issuecomment-840045326 , I would suggest we don't do anything specific for this table.